internal/app/wwctl/container/exec/main.go

```diff
@@ -143,7 +147,7 @@ func getTime(path string) time.Time {
 		return time.Time{}
 	} else {
 		unixStat := fileStat.Sys().(*syscall.Stat_t)
-		return time.Unix(int64(unixStat.Ctim.Sec), int64(unixStat.Ctim.Nsec))
+		return time.Unix(int64(unixStat.Mtim.Sec), int64(unixStat.Mtim.Nsec))
```

Comment:
I agree that mtime is probably the better time stat here; but now I'm realizing that my bug report wasn't true in the first place. I'll comment further in #1344.